src/utils.h

```diff
@@ -44,3 +44,41 @@ inline float bytes_to_float(uint8_t* address){
   f = u.float_bytes;
   return(f);
 }
+
+/**
+*  \brief Convert a long to an array of bytes.
+*
+*  \param address the memory location where the long will be written as bytes
+*  \param long_val the number which will be written to bytes
+*  \return i the length of the byte string (should always be 8)
+*
+*/
+
+inline uint8_t long_to_bytes(uint8_t* address, long long_val){
+  union {
+    long long_bytes;
+    uint8_t long_array[8];
```

Comment:
Replace these `8`s with `LONG_SIZE` and these functions are ready to go. Looks good!